pkg/filepiece/fpiece.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package fpiece
```

Comment:
I noticed that the package folder is `filepiece`, but the package name is `fpiece`. How about picking the same name for both?